lib/sorcery/model/temporary_token.rb

```diff
@@ -12,7 +12,7 @@ def self.included(base)
 
       # Random code, used for salt and temporary tokens.
       def self.generate_random_token
-        SecureRandom.base64(15).tr('+/=lIO0', 'pqrsxyz')
+        SecureRandom.urlsafe_base64(15).tr('lIO0', 'sxyz')
```

Comment:
When we drop rails 4.x supports, we might be able to deprecate it with `has_secure_token`.